module/userProjects/helper.js

```diff
@@ -2280,6 +2326,439 @@ module.exports = class UserProjectsHelper {
+                    projectId : (data._id).toString(),
+                    projectName : data.title,
+                    programId : (certificateTemplateDetails[0].programId).toString(),
+                    programName : ( data.programInformation && data.programInformation.name ) ? data.programInformation.name : "",
+                    solutionId : (certificateTemplateDetails[0].solutionId).toString(),
+                    solutionName : ( data.solutionInformation && data.solutionInformation.name ) ? data.solutionInformation.name :  "",
+                    completedDate : data.completedDate
+                };
+                return resolve(certificateData);
+
+            } catch (error) {
+                return resolve({
+                    success: false,
+                    message: error.message
+
+                });
+            }
+        })
+    }    
+
+    /**
+     * call sunbird-RC for certificate creation.
+     * @method
+     * @name createCertificate 
+     * @param {Object} certificateData - payload for certificate creation data.
+     * @returns {Boolean} certificate creation status.
+    */
+
+    static createCertificate(certificateData) {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                const certificateDetails = await certificateService.createCertificate( certificateData );      
+                if ( !certificateDetails.success || !certificateDetails.data || !certificateDetails.data.ProjectCertificate ) {
+                    throw {
+                        message:  CONSTANTS.apiResponses.CERTIFICATE_GENERATION_FAILED
+                    };
+                }
+            
+                let updateObject = {
+                    "$set" : {}
+                };
+
+                //  if transaction id is present.
+                if ( certificateDetails.data.ProjectCertificate.transactionId &&
+                    certificateDetails.data.ProjectCertificate.transactionId !== "" 
+                ) {
+                    let transactionIdvalue = certificateDetails.data.ProjectCertificate.transactionId;
+                    const first2 = transactionIdvalue.slice(0, 2);
+                    
+                    if ( first2 === "1-" ) {
+                        transactionIdvalue = transactionIdvalue.split(/1-(.*)/s)
+                        updateObject["$set"]["certificate.transactionId"] = transactionIdvalue[1];
+                    } else {
+                        updateObject["$set"]["certificate.transactionId"] = transactionIdvalue;
+                    }
+                        
+                }
+
+                // update project details certificate details
+                if ( certificateDetails.data.ProjectCertificate.osid &&
+                    certificateDetails.data.ProjectCertificate.osid !== "" 
+                ) {
+                    updateObject["$set"]["certificate.osid"] = certificateDetails.data.ProjectCertificate.osid;
```

Comment:
@VISHNUDAS-tunerlabs please create a readme and put in the project doc for the steps to create this entity in RC. Link confluence doc. See if ProjectCertificate can be dynamic and be accepted via env. default can be ProjectCertificate